src/business/CiaAerea.java

```diff
@@ -0,0 +1,88 @@
+package business;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class CiaAerea {
+    private UUID codigo;
+    private String nome;
+    private String razaoSocial;
+    private String cnpj;
+    private ArrayList<Voo> voos;
+
+    public CiaAerea() { }
+
+    public CiaAerea(String nome, String razaoSocial, String cnpj) {
+        this.codigo = UUID.randomUUID();
+        this.nome = nome;
+        this.razaoSocial = razaoSocial;
+        this.cnpj = cnpj;
+    }
+
+    public UUID getCodigo() {
+        return codigo;
+    }
+
+    public String getNome() {
+        return nome;
+    }
+
+    public void setNome(String nome) {
+        this.nome = nome;
+    }
+
+    public String getRazaoSocial() {
+        return razaoSocial;
+    }
+
+    public void setRazaoSocial(String razaoSocial) {
+        this.razaoSocial = razaoSocial;
+    }
+
+    public String getCnpj() {
+        return cnpj;
+    }
+
+    public void setCnpj(String cnpj) {
+        this.cnpj = cnpj;
+    }
+
+    public ArrayList<Voo> getVoos() {
+        return voos;
+    }
+
+    public void setVoos(ArrayList<Voo> voos) {
+        this.voos = voos;
+    }
+
+    public boolean adicionarVoo(Voo voo) {
+        this.voos.add(voo);
```

Comment:
public boolean adicionarVoo(Aeroporto localPartida, Aeroporto localChegada) {
     return this.voos.add(new Voo(this, localPartida, localChegada));
}